sdk/containerregistry/azure-containerregistry/azure/containerregistry/__init__.py

```diff
@@ -17,7 +17,10 @@
     ArtifactManifestProperties,
     RepositoryProperties,
     ArtifactTagProperties,
+    DownloadManifestResult,
 )
+from ._download_stream import DownloadBlobStream
+from .aio._async_download_stream import AsyncDownloadBlobStream
```

Comment:
This should be exposed in `aio.__init__.py`, not here.

Reply:
Updated